lighthouse-core/test/gather/fetcher-test.js

```diff
@@ -4,26 +4,13 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
+import {Fetcher} from '../../gather/fetcher.js';
+import {Driver} from '../../gather/driver.js';
 import {Connection} from '../../gather/connections/connection.js';
 import {fnAny, mockCommands} from '../test-utils.js';
 
 const {createMockSendCommandFn} = mockCommands;
 
-// Some imports needs to be done dynamically, so that their dependencies will be mocked.
```

Comment:
this was 100% unnecessary, and actually triggers a bug in testdouble I'm in the middle of fixing